lib/gui/app/components/featured-project/featured-project.jsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const React = require('react')
+const propTypes = require('prop-types')
+const SafeWebview = require('../safe-webview/safe-webview.jsx')
+const settings = require('../../models/settings')
+const analytics = require('../../modules/analytics')
+const endpoint = null
+
+class FeaturedProject extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      endpoint
+    }
+  }
+
+  componentDidMount () {
+    return settings.load()
+      .then(() => {
+        if (settings.has('featuredProjectEndpoint')) {
+          this.setState({ endpoint: settings.get('featuredProjectEndpoint') })
+        } else {
+          this.setState({ endpoint: 'https://assets.balena.io/etcher-featured/index.html' })
+        }
+      })
+      .catch(analytics.logException)
+  }
+
+  render () {
+    return (this.state.endpoint) ? (
```

Comment:
Because it is initialized with `null` since the settings need to be loaded async (aka after the first render), in case one wants to specify a different endpoint like we did for development